ios/browser/api/ads/brave_ads.mm

```diff
@@ -310,7 +310,11 @@ - (void)shutdown:(nullable void (^)())completion {
           delete self->adsClient;
         }
         if (self->adsDatabase != nil) {
-          delete self->adsDatabase;
+          self->databaseQueue->PostTask(
+              FROM_HERE,
+              base::BindOnce(
+                  [](brave_ads::Database* database) { delete database; },
```

Comment:
Should this be in a separate ads PR?

Reply:
@zenparsing Not 100% sure if this occurred outside of this PR, but definitely happened while I was testing my changes in this branch so I just tossed it in here as well. We can take it out and have it in a separate PR entirely if we want

Reply:
No strong opinion here, just checking!